lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,202 @@
+
+[.lesson-point]
+Simpson's Paradox: visible trends in sub-groups _disappear_ or even _reverse_ when the groups are combined.
+
+Normally we think that "more data means more power!", and that the more data we include in our sample the more clearly we'll see any potential correlations. But in certain circumstances - like in our Covid dataset! - the correlations in our sub-groups cancel each other out when we put the groups together! This is called Simpson's Paradox, and you can learn more at @link{https://en.wikipedia.org/wiki/Simpson%27s_paradox, Wikipedia}.
+
+[.lesson-instruction]
+- We need to break this dataset up into _grouped samples_, so that all of the data for Rhode Island is in one table, all of the data for Maine is in another, etc.
+- Working in pairs or small groups, complete the rest of of @printable-exercise{exploring-covid.adoc}.
+
+How does the `filter` function work?
+
+__The `filter` function consumes a Table and **a helper function!** The helper function is used on every Row of the Table, producing true or false. The `filter` function takes all the Rows for which the helper produced true, and combines them all into a new table.__
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+
+|
+@span{.title}{More Practice with Filtering}
+
+While filtering is introduced in this lesson, the primary goal is about exponential functions. If your students need more practice with filtering - or wish to filter their own datasets! - we recommend checking out the @lesson-link{table-functions} lesson.
+|===
+
+We can use `filter` to split our dataset into multiple sub-groups, and then fit models to each sample.
+
+[.lesson-instruction]
+Create helper functions to filter at least three other states, then use them to define the Covid tables for each state.
+
+=== Common Misconceptions
+
+It's extremely common for students to think that filtering a table *changes the original table*. This is NOT how it works in Pyret! Instead, the `filter` function always produces a _new_ table, containing only the Rows for which the supplied function evaluates to `true`
+
+=== Synthesize
+- In what other situations would it be useful to filter a dataset?
+- Can you think of other examples where Simpson's Paradox might arise?
+
+
+== Looking for Patterns @duration{15 minutes}
+
+=== Overview
+
+Students explore their newly-filtered `MA-covid` dataset, trying to fit different kinds of models to it. This section makes heavy use of @link{https://www.Desmos.com, Desmos}, which is an excellent tool for open-ended experimentation.
+
+=== Launch
+
+Make a scatter-plot showing the Covid infection rate for Massachusetts. What kind of model do you think would fit this best?
+
+=== Investigate
+
+[.lesson-instruction]
+- Open @printable-exercise{graphing-models.adoc} and @online-exercise{https://www.desmos.com/calculator/09kyr4xlct, MA Covid Spread - Linear}.
+- Complete the *first* section on @printable-exercise{graphing-models.adoc}.
+
```

Comment:
did you wanna hide the dataset in desmos? I commented a link to a 30 sec youtube video showing one way of doing it.